### DISCUSSION TITLE
v20.6.0

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3928] [Releases]
A solid bugfix release this 20.6.0, with some accessibility improvements to the RWD theme too, which will improve the lighthouse scores that everybody loves ;-)

## Changelog
* Fixed zIndex to avoid conflicts between TinyMCE and OpenMage's overlays in the CMS section by @fballiano in https://github.com/OpenMage/magento-lts/pull/3851
* Fixed default width for input/textarea in OpenMage backend theme by @fballiano in https://github.com/OpenMage/magento-lts/pull/3850
* Improved fields dependence (show/hide) in backend's config section by @luigifab in https://github.com/OpenMage/magento-lts/pull/2150
* Varien_Directory_Collection::getLastDir(): "$this" can no longer be used in a plain function or method, since PHP 7.1 by @fballiano in https://github.com/OpenMage/magento-lts/pull/3823
* Fixed null error on hash_equal in Mage_Oauth_Model_Server by @kiatng in https://github.com/OpenMage/magento-lts/pull/3870
* Fixed row click issue in Category Products tab by @ADDISON74 in https://github.com/OpenMage/magento-lts/pull/3866
* Fixed null deprecation in lib/Varien/Filter/Template.php by @kiatng in https://github.com/OpenMage/magento-lts/pull/3879
* Fixed row click issue in product's upsell/crosssell/related grids by @ADDISON74 in https://github.com/OpenMage/magento-lts/pull/3865
* Added PHP8.3 to supported list by @fballiano in https://github.com/OpenMage/magento-lts/pull/3887
* Added VAT related attributes to salesOrderAddressEntity webservices entity by @empiricompany in https://github.com/OpenMage/magento-lts/pull/3885
* RWD: enhanced contrast on a few colors to fix accessibility issues by @fballiano in https://github.com/OpenMage/magento-lts/pull/3911
* Fixed no data shown for products with recurring profiles by @fballiano in https://github.com/OpenMage/magento-lts/pull/3910
* RWD: accessibility improvements: made the red color a bit darker and added titles to important links by @fballiano in https://github.com/OpenMage/magento-lts/pull/3912
* Rewrote getOpenMageVersion() to be faster by @fballiano in https://github.com/OpenMage/magento-lts/pull/3875
* Removed unused variables from Mage_Api_Model_Server_Handler_Abstract by @fballiano in https://github.com/OpenMage/magento-lts/pull/3862
* Removed @deprecation on Mage::getBlockSingleton() by @kiatng in https://github.com/OpenMage/magento-lts/pull/3835
* Fixed null parameter warning in Mage_Catalog_Model_Layer_Filter_Attribute by @fballiano in https://github.com/OpenMage/magento-lts/pull/3926
* Bump phpseclib/phpseclib from 3.0.35 to 3.0.37 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3869
* Bump phpseclib/mcrypt_compat from 2.0.5 to 2.0.6 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3868
* Bump friendsofphp/php-cs-fixer from 3.50.0 to 3.51.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3867
* Bump tj-actions/changed-files from 42 to 43 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3882
* Bump friendsofphp/php-cs-fixer from 3.51.0 to 3.52.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3895
* Bump phpunit/phpunit from 9.6.17 to 9.6.18 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3896
* Bump squizlabs/php_codesniffer from 3.9.0 to 3.9.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3919
* Bump tj-actions/changed-files from 43 to 44 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3918
* Bump phpstan/phpstan from 1.10.59 to 1.10.66 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3880

**Full Changelog**: https://github.com/OpenMage/magento-lts/compare/v20.5.0...v20.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMage/magento-lts/releases/tag/v20.6.0'>v20.6.0</a>.</em>

**Comment by akunzai** [comment]
Hi @fballiano, The `openmage-v20.6.0.zip` was missing under release assets

**Reply by fballiano** [reply]
mmm I've no idea why the release workflow didn't run but I've manually generated the zip and uploaded to the release, thanks for reporting this problem :-)
